build/packages.targets

```diff
@@ -8,7 +8,7 @@
         <VSServicesVersion>16.153.0</VSServicesVersion>
         <VSThreadingVersion>16.5.126</VSThreadingVersion>
         <!-- TODO - remove this when temporary patching is no longer necessary. See https://github.com/NuGet/Home/issues/8508 -->
```

Comment:
Is this still necessary per the above comment?

Reply:
The comment may not be the accurate..
So my understanding is:
 the temporary patching is for our dotnet integration test before inserting into dotnet.
Once we inserted into dotnet, dotnet will automatically update the deps.json files if they retarget the netcoreapp5.0 version of our code. 
Then before the second insertion, we can remove the temporary patching.
Is that correct? If yes, we have to keep temprorary patching until we finish the first insertion, right?

Reply:
That's fine but the issue is closed.

You should re-open the issue if it's still relevant.

Reply:
See https://github.com/NuGet/Home/issues/8508.

Reply:
Thanks Nikolche! Updated the link to https://github.com/nuget/home/issues/8952